camayoc/tests/qpc/ui/test_credentials.py

```diff
@@ -9,21 +9,55 @@
 :testtype: functional
 :upstream: yes
 """
-from pathlib import Path
+import random
 
 import pytest
 
-from .utils import check_auth_type
-from .utils import create_credential
-from .utils import delete_credential
-from .utils import edit_credential
-from camayoc import utils
-
-SOURCE_TYPES = ["Network", "Satellite", "VCenter"]
+from camayoc.qpc_models import Credential
+from camayoc.ui import Client
+from camayoc.ui import data_factories
+from camayoc.ui.data_factories import AddCredentialDTOFactory
```

Comment:
I think: either import all the classes from `data_factories` and use them directly (as in lower line here), or import `data_factories` and use classes through it (as in upper line here). No point in doing the mix of things.

Reply:
Right, I choose to keep `import data_dactories` and use it in the rest of the code as `data_factories.SSHNetworkCredentialFormDTOFactory`.